src/components/action-bar/action-bar.scss

```diff
@@ -1,10 +1,18 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-action-bar-expanded-max-width: optionally specify the expanded max width of the action bar when in "vertical" layout.
+ */
+
 :host {
   @extend %component-host;
   @apply pointer-events-auto
     inline-flex
     flex-col
     self-stretch;
-  max-inline-size: 15vw;
+  --calcite-action-bar-expanded-max-width: auto;
```

Comment:
Is the reason this is being set to `auto` by default so that it has a less opinionated starting value?

Reply:
Correct.

Reply:
https://github.com/Esri/calcite-components/issues/2380#issuecomment-1192768235